app/screens/apps_form/apps_form_component.tsx

```diff
@@ -126,18 +126,9 @@ function AppsFormComponent({
     const theme = useTheme();
     const style = getStyleFromTheme(theme);
 
-    const onHandleSubmit = useCallback(() => {
-        if (!submitting) {
-            handleSubmit();
-        }
-    }, [serverUrl, componentId, submitting]);
-
-    useNavButtonPressed(CLOSE_BUTTON_ID, componentId, close, [close]);
-    useNavButtonPressed(SUBMIT_BUTTON_ID, componentId, onHandleSubmit, [onHandleSubmit]);
-
     useDidUpdate(() => {
         dispatchValues({elements: form.fields});
-    }, [form]);
+    }, [form, dispatchValues]);
```

Comment:
There is no need to depend on `dispatchValues` since the dispatch function should not change between renders.